docs/dev/swagger-source.json

```diff
@@ -3832,7 +4413,883 @@
 					},
 					"403": {
 						"x-summary": "Request Unauthorized",
-						"description": "You do not have permission to access statistics for this journal or press.",
+						"description": "You do not have permission to access statistics for this journal.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					},
+					"404": {
+						"x-summary": "Request Not Found",
+						"description": "The requested resource was not found.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					}
+				}
+			}
+		},
+		"/stats/publications": {
+			"get": {
+				"tags": [
+					"Stats / Publications"
+				],
+				"summary": "Get a list of publications by total visits.",
+				"description": "Returns a list of publications ordered by their total abstract views. Includes total views by abstract, all galleys, PDF galleys, HTML galleys and other galleys.",
```

Comment:
> Returns a list of publications ordered by the number of views of the publication landing page.

Reply:
@NateWr, I think my formulation here was wrong, i.e. somehow 'abstract' sneaked in there :-\ They are ordered by total views which means abstract + all galleys (PDF, HTML and other) views. So maybe:
"Returns a list of publications ordered by the total number of views."?

Reply:
Good!